src/helpers/getCurrenciesValues.ts

```diff
@@ -0,0 +1,11 @@
+import { Currencies } from '../types/CountriesTypes';
+
+export const getCurrenciesValues = (currencies:Currencies) => {
+  const currenciesKeys = Object.keys(currencies || {});
```

Comment:
Lo mismo pasa aquí: Los types indican que el objeto puede que venga vacío o con una propiedad, por ende sólo le puedes referenciar los currencies:

```suggestion
  const currenciesKeys = Object.keys(currencies);
```

Pero si TS te marcó un error y esa fue la forma de mitigarlo, estoy OK 👌🏻